public/pages/Correlations/components/FilterGroup.tsx

```diff
@@ -54,7 +70,35 @@ export const FilterGroup: React.FC<LogTypeFilterGroupProps> = ({ groupName, item
           checked: 'on',
         };
     }
+    const chilIds = newItems[index].childOptionIds;
+    if (chilIds) {
+      newItems.forEach((item) => {
+        if (chilIds.has(item.id)) {
+          item.checked = newItems[index].checked;
+        }
+      });
+    }
+
+    setItems(newItems);
+    setShowActiveFilters(true);
+  }
 
+  function toggleAll(state: 'on' | undefined) {
```

Comment:
Not blocking: We may want to take a follow-up item to add a constant containing an exhaustive list of `state`s for this context.

Reply:
It's either `on` or undefined. Also since it is local to this function I don't create a separate type